prime-router/src/main/kotlin/azure/HttpUtilities.kt

```diff
@@ -210,7 +210,7 @@ class HttpUtilities {
          * Do a variety of checks on payload size.
          * Returns a Pair (http error code, human readable error message)
          */
-        fun payloadSizeCheck(request: HttpRequestMessage<String?>) {
+        fun payloadSizeCheck(request: HttpRequestMessage<String?>, maxPayloadSize: Long = PAYLOAD_MAX_BYTES) {
```

Comment:
making `PAYLOAD_MAX_BYTES` configurable for whoever needs to call this. it defaults to the Report.kt constant